OpenRA.Mods.Common/Widgets/Logic/ColorPickerLogic.cs

```diff
@@ -63,11 +66,170 @@ public ColorPickerLogic(Widget widget, ModData modData, World world, HSLColor in
 
 			hueSlider.Value = initialColor.H / 255f;
 			onChange(mixer.Color);
-		}
-
-		public static void ShowColorDropDown(DropDownButtonWidget color, ColorPreviewManagerWidget preview, World world)
-		{
-			Action onExit = () =>
+            
+            // Setup the mod team preset colors
+            int maxCustomColors = 10;
+
+            var clearCustomPaletteButton = widget.GetOrNull<ButtonWidget>("CLEAR_CUSTOM_BUTTON");
+		    if (clearCustomPaletteButton != null)
```

Comment:
Here is a reason to use tabs; the indentation for `if` here does not lined up with the rest.